tests/integration/acme-tester/lib/charms/acme_client_operator/v0/acme_client.py

```diff
@@ -0,0 +1,211 @@
+# Copyright 2021 Canonical Ltd.
```

Comment:
I am unsure we should have this file committed, I think `tox` should handle copying the library for testing, I feel like this will be too easy to forget to update this duplicate.

Reply:
You are right, done